task_7/models.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from typing import List, Optional, Literal
+
+from sqlalchemy import func, ForeignKey, Column, false, Enum
+from sqlalchemy.orm import Mapped, mapped_column, relationship, Relationship
```

Comment:
```suggestion
from sqlalchemy.orm import Mapped, mapped_column, relationship
```

remove un used import statements like this